clojure2d support and demos added

genmeblog opened the pull request:
Migrated clojure2d examples to the library.
Haven't touched README since I didn't know how you want to promote information about this addition.

genmeblog committed:
clojure2d support and demos added

plexus commented:
Fantastic! Really appreciate having this. I've used bits of Clojure2D in [Witchcraft](https://github.com/lambdaisland/witchcraft) (just color functions IIRC), but this is a great excuse to play around with it some more.

I'll update the README, I think Quil and Clojure2D can be presented as two options with equal level of support.

genmeblog commented:
Lovely! I revisited whole `color` namespace In 1.4.4, fixed some bugs, added more stuff (more delta-... functions for example) and started to make thorough documentation (here: https://clojure2d.github.io/clojure2d/docs/notebooks/index.html#/notebooks/color.clj)
